resources/views/artists/show.blade.php

```diff
@@ -46,6 +46,14 @@
                     <h1>{{ $artist->name }}</h1>
 
                     {!! markdown($artist->description) !!}
+
+                    @if ($artist->video_url !== null)
+                        <div class="artist__video">
+                            <div class="u-embed-wide">
+                                <iframe src="{{ $artist->video_url }}" allowfullscreen></iframe>
+                            </div>
```

Comment:
If they're always self-hosted then just use `video`; putting video url into iframe also clips the controls on Safari